modmain.lua

```diff
@@ -42,6 +42,13 @@ local COMPACTSEASONS = SEASONOPTIONS == "Compact"
 local MICROSEASONS = SEASONOPTIONS == "Micro"
 local HUDSCALEFACTOR = GetModConfigData("HUDSCALEFACTOR")*.01
 
+if SHOWNAUGHTINESS then
+	if DST and not GLOBAL.KnownModIndex:IsModEnabled("workshop-2189004162") then -- shouldnt show naughtiness if this isnt here to provide it
+		SHOWNAUGHTINESS = false
+	end
+end
+	
+

```

Comment:
Can you roll this into the CHECK_MODS section below? It'll make it easier to extend this if there are other mods that add naughtiness, and also checks for it two ways. If I remember correctly, KnownModIndex:IsModEnabled only gets mods that are already loaded.

Reply:
Done